### PR TITLE
[v7r3] Fix: reintroducing Staging to Checking transition

### PR DESCRIPTION
**Pull request by michmx**
BEGINRELEASENOTES

*WorkloadManagement
FIX: reintroduce 'Staging' to 'Checking' transition in JobsStateMachine

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py [hunk 1]
```diff
@@ -95,8 +95,8 @@ def __init__(self, state):
             RESCHEDULED: State(6, [WAITING, RECEIVED, DELETED, FAILED], defState=RESCHEDULED),
             MATCHED: State(5, [RUNNING, FAILED, RESCHEDULED, KILLED], defState=MATCHED),
             WAITING: State(4, [MATCHED, RESCHEDULED, DELETED], defState=WAITING),
-            STAGING: State(3, [WAITING, FAILED, KILLED], defState=STAGING),
-            SCOUTING: State(2, [CHECKING, FAILED, STALLED, KILLED], defState=SCOUTING),
+            STAGING: State(3, [CHECKING, WAITING, FAILED, KILLED], defState=STAGING),
```
Comment (andresailer) [inline src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py]: Does this state ID (3) have to be unique?

Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py]: No, but it better be.

Comment (michmx) [reply src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py]: Ok, I will re-enumerate all states

Comment (michmx) [reply src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py]: I have reassigned IDs to the states. Please check to confirm that you agree.

**Review by andresailer — APPROVED** [review]
Looks good to me!

**Pull request merged**